include/groonga/util.h

```diff
@@ -23,6 +23,30 @@
 extern "C" {
 #endif
 
+/**
+ * \brief Inspect specified object and produce a textual representation.
+ *
+ * \since 4.0.8
```

Comment:
Probably, this is wrong.

`grn_inspect()` added since 0.7.0 by the following commit.
ref: https://github.com/groonga/groonga/commit/675589c661d868a9a442f03d5df47776b794d477

Reply:
fix [bc5ccff](https://github.com/groonga/groonga/pull/2328/commits/bc5ccff767c17600b354fe854331bc5b9f4cdcde): Thanks!